lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ /**
+  * @fileoverview Checks to see if images are displayed only outside of the viewport.
+  */
+'use strict';
+
+const Audit = require('./byte-efficiency-audit');
+const TTIAudit = require('../time-to-interactive');
```

Comment:
i'm with ya, @paulirish not a fan of getting these values into computed artifact though 😩